components/Results.js

```diff
@@ -46,26 +47,55 @@ export default class Results extends React.Component {
           </View>
         );
```

Comment:
I found a couple of edge cases: 
If you enter 1 - it should go in this case.

Reply:
Check where empty goes - if we enter nothing it should go here but goes to "No information"